packages/frontend/src/lib/Badge.svelte

```diff
@@ -2,13 +2,25 @@
 import type { IconDefinition } from '@fortawesome/free-regular-svg-icons';
 import Fa from 'svelte-fa';
 
-export let icon: IconDefinition;
-export let content: string;
-export let background: string = 'bg-[var(--pd-label-bg)]';
+interface Props {
+  icon?: IconDefinition;
+  content: string;
+  backgroundColor?: string;
```

Comment:
Do we really need to ue backgroundColor? I feel like this should be included in the `class` prop?